.github/workflows/manylinux.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ['3.6', '3.7', '3.8']
+        python-version: ['2.7']
```

Comment:
so you're dropping CI support for python3 on those platforms ?

Reply:
Hi !

Of course not, It is a debugging commit. I would rather fix the unit tests for Python 2.7 before reactivating them for Python 3 to avoid using more computing power than necessary.